Packs/HashiCorp-Vault/Integrations/HashiCorpVault/README.md

```diff
@@ -404,6 +404,67 @@ Create a new authentication token.
 #### Command example
 ```!hashicorp-create-token display_name=token explicit_max_ttl=3600 renewable=false```
 
+
+### hashicorp-generate-role-secret
+***
+Generates and issues a new SecretID on an existing AppRole.
+
+#### Base Command
+
+`hashicorp-generate-role-secret`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| role_name | The name of the AppRole. | Required |
+| meta_data | Metadata to be tied to the SecretID. | Optional |
+| cidr_list | Comma separated string or list of CIDR blocks enforcing secret IDs to be used from specific set of IP addresses. | Optional |
+| token_bound_cidrs | Comma-separated string or list of CIDR blocks. | Optional |
+| num_uses | Number of times this SecretID can be used, after which the SecretID expires. A value of zero will allow unlimited uses. | Optional |
```

Comment:
Number of times the SecretID can be used, after which the SecretID expires. A value of zero will allow unlimited uses.